### PR TITLE
doc: update manual to include IP address form field

### PR DESCRIPTION
**Pull request by marharyta**
# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [ ] Link to issue
- [ ] Note if PR is on top of other PR
- [ ] Note if related change in rems-deploy repo
- [ ] Consider adding screenshots for ease of review

## Backwards compatibility
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible _or_ have migrations
- [ ] Config is backwards compatible

## Documentation
- [ ] Update changelog if necessary
- [ ] API is documented and shows up in Swagger UI
- [ ] Components are added to guide page
- [ ] Update docs/ (if applicable)
- [ ] Update manual/ (if applicable)
- [ ] ADR for major architectural decisions or experiments
- [ ] New config options in config-defaults.edn

## Testing
- [ ] Complex logic is unit tested
- [ ] Valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [ ] New tasks are created for pending or remaining tasks

### CODE REVIEW

**Comment by marharyta** [comment]
Would that be it or am I missing something?

File: manual/owner.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Application forms are totally customisable. You can create different fields, suc
 **Attachment**: Applicant can add one or multiple attachments.
 **Label**: Use labels as descriptive texts if you want to give the applicant additional information, such as instructions, regarding some field.
 **Header**: Consider using headers if you want to divide the application form into sections.
+**IP address**: IP address field will validate the IP address the applicant has provided in IPv4 or IPv6 forms.
```
Comment (opqdonut) [inline manual/owner.md:63]: how about the phone number field?

Comment (opqdonut) [inline manual/owner.md:63]: the fact that private addresses are disallowed should be mentioned briefly

**Review by opqdonut — CHANGES_REQUESTED** [review]

File: manual/owner.md (hunk 1 above)
Comment (Macroz) [inline manual/owner.md:63]: Shouldn't we mention here that the IP address also can't be in a known private range?

**Review by Macroz — CHANGES_REQUESTED** [review]

File: manual/owner.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ Application forms are totally customisable. You can create different fields, suc
 **Attachment**: Applicant can add one or multiple attachments.
 **Label**: Use labels as descriptive texts if you want to give the applicant additional information, such as instructions, regarding some field.
 **Header**: Consider using headers if you want to divide the application form into sections.
+**IP address**: IP address field will validate the IP address the applicant has provided in IPv4 or IPv6 forms. IP address must not be withing the known private range.
```
Comment (opqdonut) [inline manual/owner.md:63]: typo: withing

File: manual/owner.md (hunk 1 above)
Comment (opqdonut) [reply manual/owner.md]: separate PR I hear

**Review by opqdonut — APPROVED** [review]

**Pull request merged**